docs/build/guides/conventions/soroban-contract-init-template.mdx

```diff
@@ -0,0 +1,432 @@
+    );
+    rmSync(filePath);
+  } catch (err) {
+    console.error(`Error while deleting Files \n ${err}`);
+  }
+});
+
+//Build Wasm Contract Files
+run("stellar contract build");
+
+//Deploy Contract Wasm and save the contact Ids and Aliases
+const files = globSync("target/wasm32-unknown-unknown/release/*.wasm");
+let alias, contractId;
+console.log(files);
+files.forEach((file) => {
+  alias = path.basename(file).split(".")[0];
+  console.log(`Deploying Contract ${alias}`);
+  contractId = run(
+    `stellar contract deploy --network ${process.env.NETWORK} --source ${process.env.ACCOUNT} --wasm ${file} --alias ${alias}`,
+  );
+  //save contractId and alias to contracts Object
+  contractsObject[alias] = {
+    alias,
+    id: contractId,
+  };
+});
+
+//Create a json File to store the contract name as key and contract id and contract alias contained in an object as value
+const contractsJSONPath = "./contracts.json";
+const contractsJSONData = JSON.stringify(contractsObject);
+
+try {
+  writeFileSync(contractsJSONPath, contractsJSONData);
+  console.log(`JSON file has been saved to ${contractsJSONPath}`);
+} catch (err) {
+  console.error("Error writing JSON file: ", err);
+}
+
+//Generate Contract Typescript Bindings
+let filename, output_dir;
+files.forEach((file) => {
+  filename = path.basename(file).split(".")[0];
+  output_dir = `./packages/${filename}`;
+  run(
+    `stellar contract bindings typescript --output-dir ${output_dir} --network ${process.env.NETWORK} --contract-id ${contractsObject[alias]["id"]}  --wasm ${file} --overwrite`,
+  );
+});
+```
+
+## Modifying `package.json` file
+
+After saving the `initialize.js` script, let's modify our `package.json` file to create a new entry under scripts to run the code to work on the contract files.
+
+Let's also modify the file to create a new workspace in the `package.json` to point to the path where the the TypeScript bindings are generated
+
+```json
+"scripts": {
+    "setup": "node initialize.js",
+  },
+
+ "workspace": ["./packages/*"],
+```
+
+## Running the script
+
+Now that we have a complete script and a frontend, we should try to run the script. However, because the script is dependent on contract files we need to have a contract project ready, otherwise the script will fail and result in an error. To solve this issue we have to either copy the whole project contents and paste them to an existing Stellar contract project directory or upload the template file to a repo then create a new project and setting the 'f' parameter to point to the hosted repository of the template.
+
+### Setting up the project
+
+A community member has hosted a [public repository of the React template](https://github.com/CaptainPrinz/soroban-react-template) available for use. The following command implements the second option:
+
+```bash
+stellar contract init `hello_contract` -w hello -f https://github.com/CaptainPrinz/soroban-react-template.git
```

Comment:
```suggestion
stellar contract init hello_contract -f https://github.com/CaptainPrinz/soroban-react-template.git
```
The stellar cli (as of `v21.2.0` this morning) will fail if you run this command as-is. The project directory (hello_contract) needs to _not_ be in back-ticks, and `-w hello` is unnecessary, as the hello world contract is included by default. The cli will give an error about it being specified, too.

Reply:
fixed